wanda-docker-compose.yaml

```
@@ -0,0 +1,33 @@
+version: "3"
+
+services:
+
+  postgres:
+    volumes:
+      - ./container_fixtures/postgres/init_wanda_db.sh:/docker-entrypoint-initdb.d/init_wanda_db.sh:ro
+
+  checks: 
+      image: ghcr.io/trento-project/checks:rolling
+      volumes:
+        - trento-checks:/usr/share/trento/checks
+
+  wanda:
+    image: ghcr.io/trento-project/trento-wanda:demo
+    environment:
+      DATABASE_URL: ecto://postgres:postgres@postgres/postgres
```

Let's use the username/password/db created for wanda

```suggestion
      DATABASE_URL: ecto://wanda_user:wanda_pass@postgres/wanda_db
```